js/explore/largeRepos/sunburst_licenses.js

```diff
@@ -0,0 +1,331 @@
+            .tickFormat(d => {
+                return options[Math.round(d)];
+            })
+            .ticks(3)
+            .on('onchange', val => {
+                data = reformatData(obj, val);
+                root = partition(data);
+                root.each(d => d.current = d);
+                path.data(root.descendants().slice(1)).join(enter => enter, update => {
+                    update.attr('fill', d => {
+                                while (d.depth > 1) {
+                                    d = d.parent;
+                                }
+                                return colors(d.parent.children.indexOf(d));
+                            })
+                            .attr('fill-opacity', 0.9)
+                            .attr('d', d => {
+                                if (lastClickedObject.depth == 0) {
+                                    return arc(d.current);
+                                } else {
+                                    let o = root.children.filter(d => d.data.name == lastClickedObject.data.name)[0];
+                                    root.each(d => d.current = {
+                                        x0: Math.max(0, Math.min(1, (d.x0 - o.x0) / (o.x1 - o.x0))) * 2 * Math.PI,
+                                        x1: Math.max(0, Math.min(1, (d.x1 - o.x0) / (o.x1 - o.x0))) * 2 * Math.PI,
+                                        y0: Math.max(0, d.y0 - o.depth),
+                                        y1: Math.max(0, d.y1 - o.depth)
+                                    });
+                                    return arc(d.current);
+                                }
+                            });
+                    return update;
+                }, exit => exit);
+                label.data(root.descendants().slice(1))
+                    .join('text')
+                        .attr('dy', '0.35em')
+                        .attr('fill-opacity', d => +labelVisible(d.current))
+                        .attr('transform', d => labelTransform(d.current))
+                        .text(d => {
+                            let name = d.data.name;
+                            if (d.height > 0) {
+                                return name;
+                            } else {
+                                name = name.split('/')[1];
+                                if(name.length > 20) {
+                                    return '...';
+                                } else {
+                                    return name;
+                                }
+                            }
+                        });
+                path.selectAll('title').remove();
+                path.append('title').text(d => d.data.name);
+            });
+    
+        // Creates option slider
+        /*chart.append('g')
+            .attr('transform', `translate(${margin.left},${margin.top / 2})`)
+            .attr('id', 'licenseSlider')
+            .call(slider);*/
```

Comment:
I think the option slider would still be useful for the top 10. We are still showing the "Top 10 Repos by Stars", but it still makes sense to change their relative proportions based on other factors.
```suggestion
        // Creates option slider
        chart.append('g')
            .attr('transform', `translate(${margin.left},${margin.top / 2})`)
            .attr('id', 'licenseSlider')
            .call(slider);
```

Reply:
Alright, sounds good. I didn't know how it would look to someone else, so good to know.